package.json

```diff
@@ -19,7 +19,9 @@
     "install-packages": "lerna exec -- yarn"
   },
   "dependencies": {
-    "dooboolab-welcome": "^1.2.0"
+    "d3": "^5.16.0",
+    "dooboolab-welcome": "^1.2.0",
+    "react-native-svg": "^12.1.0"
```

Comment:
Please remove the dependencies in the root! Only `dooboolab-welcome` is permitted here.